Documentation/technote-ocsp-cert-revocation.md

```diff
@@ -0,0 +1,290 @@
+[![DAML logo](https://daml.com/wp-content/uploads/2020/03/logo.png)](https://www.daml.com)
+
+[![Download](https://img.shields.io/github/release/digital-asset/daml.svg?label=Download)](https://docs.daml.com/getting-started/installation.html)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/digital-asset/daml/blob/master/LICENSE)
+
+# TechNote: Certificate Revocation
+
+Digital certificates in the form of server-side TLS certificates and client-side mutual authentication certificates are used
+to authenticate and protect data in transit. Mutual authentication allows both sides to validate who they are
+connecting to and make decisions around whether this is acceptable.
+
+However digital certificates can be lost, stolen, expired, revoked, and each side may want to perform further validation. Over the
+years, many mechanisms have been developed to allow applications to validate the certificate and these have different tradeoffs.
+
+This TechNote documents the certificate revocation mechanisms, some options to enable debugging of the flows
+and some instructions on how to the use to provided test scripts and programs to test out revocation
+options.
+
+## Types of Certificate Revocation Checks
+
+Certificate Revocation mechanisms include:
+
+- **Certificate Revocation Lists (CRL)**
+  - A signed list of Certificates that have been revoked by the CA
+- **OCSP**
+  - Protocol to allow clients of the Certificate Authority to request status of specific certs
+- **OCSP Stapling**
+  - Enhancement to OCSP to allow the server to "staple" the OCSP response removing need for individual clients to check
```

Comment:
```suggestion
  - Enhancement to OCSP to allow the server to "staple" the OCSP response removing need for individual clients to check this in a separate request
```
at least afaik client still check this. the difference is how they get the information they check against.